spec/lib/insert/insert_payout_spec.rb

```diff
@@ -164,7 +164,7 @@
           expect(result.with_indifferent_access).to eq expected_result.merge(id: resulted_payout.id)
 
           empty_db_attributes = {manual: nil, scheduled: nil, failure_message: nil}
-          expect(resulted_payout.attributes.with_indifferent_access).to eq expected_result.merge(id: resulted_payout.id).merge(empty_db_attributes)
+          expect(resulted_payout).to have_attributes(expected_result.merge(id: resulted_payout.id).merge(empty_db_attributes))
```

Comment:
This isn't specific to this point as it's needed every place we would have created a Payout but one thing we still need to check for is that the publish_created method was called. The easiest way be to make sure that the new Payout has one object_event of `payout.created`

Reply:
Gotcha. Added a test to insert_payout_spec. Lmk if it covers what you were thinking